From: Martin Gräßlin <mgraesslin@kde.org>
Date: Fri, 19 Sep 2014 07:46:55 +0000
Subject: Do not install Server library
X-Git-Tag: v5.0.95
X-Git-Url: http://quickgit.kde.org/?p=kwayland.git&a=commitdiff&h=789c6729532d83e7677543e04230d9bf6e74d84f
---
Do not install Server library

It's not yet used by KWin, so we don't need to expose it yet.
This gives us more time to fine tune the API.
---


--- a/src/server/CMakeLists.txt
+++ b/src/server/CMakeLists.txt
@@ -30,17 +30,17 @@
                                                  EXPORT_NAME WaylandServer
 )
 
-install(TARGETS KF5WaylandServer EXPORT KF5WaylandTargets ${KF5_INSTALL_TARGETS_DEFAULT_ARGS})
+# install(TARGETS KF5WaylandServer EXPORT KF5WaylandTargets ${KF5_INSTALL_TARGETS_DEFAULT_ARGS})
+#
+# install(FILES
+#   ${CMAKE_CURRENT_BINARY_DIR}/kwaylandserver_export.h
+#   buffer_interface.h
+#   compositor_interface.h
+#   display.h
+#   output_interface.h
+#   seat_interface.h
+#   shell_interface.h
+#   surface_interface.h
+#   DESTINATION ${KF5_INCLUDE_INSTALL_DIR}/KWayland/Server COMPONENT Devel
+# )
 
-install(FILES
-  ${CMAKE_CURRENT_BINARY_DIR}/kwaylandserver_export.h
-  buffer_interface.h
-  compositor_interface.h
-  display.h
-  output_interface.h
-  seat_interface.h
-  shell_interface.h
-  surface_interface.h
-  DESTINATION ${KF5_INCLUDE_INSTALL_DIR}/KWayland/Server COMPONENT Devel
-)
-

